### PR TITLE
format: add appbase.io to opensearch partners page

### PR DESCRIPTION
**Pull request by siddharthlatest**
### Description

Adds appbase.io to the opensearch partners page

---

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by elfisher** [comment]
This LGTM (screenshot below). @siddharthlatest would you mind switching the branch that you are merging into to `staging`? 

image

File: _partners/appbaseio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+name: Appbase.io
+logo: '/assets/media/partners/appbase-io.png'
```
Comment (stockholmux) [inline _partners/appbaseio.md:3]: Do you have a transparent background png? It will look cleaner and more consistent.

**Review by stockholmux — CHANGES_REQUESTED** [review]
One small nit with the image.

**Comment by siddharthlatest** [comment]
@stockholmux @elfisher Thanks, have addressed both parts.

**Review by elfisher — APPROVED** [review]
LGTM

**Comment by stockholmux** [comment]
Looks great!

**Pull request merged**